README.md

```diff
@@ -6,6 +6,7 @@
 [![Nightlies](https://github.com/containerd/containerd/workflows/Nightly/badge.svg)](https://github.com/containerd/containerd/actions?query=workflow%3ANightly)
 [![Go Report Card](https://goreportcard.com/badge/github.com/containerd/containerd/v2)](https://goreportcard.com/report/github.com/containerd/containerd/v2)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1271/badge)](https://bestpractices.coreinfrastructure.org/projects/1271)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/containerd/containerd/badge)](https://api.securityscorecards.dev/projects/github.com/containerd/containerd)
```

Comment:
Clicking the link takes me to the `api.` subdomain, which is not human-readable; should that be a different subdomain? https://api.securityscorecards.dev/projects/github.com/containerd/containerd